tests/cli/init.sh

```diff
@@ -4,103 +4,103 @@ CLIENT_EXE=basecli
 SERVER_EXE=basecoin
 
```

Comment:
what is the change you made here on all the lines?  tabs to spaces?

let's get things straight with our editors to not have this issue in the future.